### DISCUSSION TITLE
.NET MAUI 8.0.92 (Service Release 9.2)

### DISCUSSION THREAD

**Discussion by rachelkang** [dotnet/maui#25300] [Announcements]
This release is now available via [NuGet.org](https://www.nuget.org/packages/Microsoft.Maui.Controls/8.0.92) and is compatible with the latest .NET 8 workload. 

```
<ItemGroup>
  <PackageReference Include="Microsoft.Maui.Controls" Version="8.0.92" />
  <PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="8.0.92" />
</ItemGroup>
```

The focus of our service releases is resolving high-impact bugs for developers upgrading their applications from Xamarin to .NET 8.

- [Release notes](https://github.com/dotnet/maui/releases/tag/8.0.92)

**Comment by b12kab** [comment]
Is 8.0.92 release only just for the Maui nuget items, such as Microsoft.Maui.Controls? No issue with getting those, but when I try getting the maui workload, it is not found:

% cat temp.json
{
"microsoft.net.sdk.maui": "8.0.92/8.0.100"
}
% sudo dotnet workload update --from-rollback-file temp.json
Welcome to .NET 8.0!
...
Workload installation failed. Rolling back installed packs...
Workload update failed: Failed to install manifest microsoft.net.sdk.maui version 8.0.92: Version 8.0.92 of package microsoft.net.sdk.maui.manifest-8.0.100 is not found in NuGet feeds https://api.nuget.org/v3/index.json"..

**Reply by nll** [reply]
The MAUI Workload and MAUI Controls are two different things. The workload have less frequent releases but do have the same version as the controls, so the versions of both will drift apart in some occasions.
 - The latest version of [MAUI Workload](https://www.nuget.org/packages/Microsoft.NET.Sdk.Maui.Manifest-8.0.100/8.0.82/#versions-body-tab) is 8.0.82
 - The latest version of [MAUI Controls](https://www.nuget.org/packages/Microsoft.Maui.Controls/8.0.92#versions-body-tab) is 8.0.92
